### PR TITLE
[rebrand] add inputs

### PR DESCRIPTION
**Pull request by DanielBalsam**
Accordion coming later today! 😊

### CODE REVIEW

File: src/pages/style-guide.js [hunk 1]
```diff
@@ -7,127 +7,165 @@ import Layout from 'components/Layout'
...
-            <h4 className={styles.label}>Typeface</h4>
-            <div className={styles.typefaceDemoGroup}>
-              <div className={styles.typefaceDemoItem}>
-                <div className={cx(styles.typefaceTitle, styles.typefaceBase)}>
-                  Aa
+const StyleGuide = () => {
+  const [inputValue, setInput] = useState('')
+
+  return (
+    <Layout className="style-guide">
```
Comment (DanielBalsam) [inline src/pages/style-guide.js:18]: just indented so I could add a state variable

File: src/pages/style-guide.js [hunk 2]
```diff
@@ -7,127 +7,165 @@ import Layout from 'components/Layout'
...
-                  IBM Plex Sans Regular
-                </span>
-              </div>
 
-              <div className={styles.typefaceDemoItem}>
-                <div className={cx(styles.typefaceTitle, styles.typefaceLight)}>
-                  Aa
+                <div className={styles.typefaceDemoItem}>
+                  <div
+                    className={cx(styles.typefaceTitle, styles.typefaceLight)}
```
Comment (DanielBalsam) [inline src/pages/style-guide.js:78]: I use prettier for auto-formatting, if this is annoying I'll turn it off for this project.

File: src/components/Accordion/Accordion.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+export const Accordion = ({ label, content }) => {
+  const [uuid] = useState(generateUUID())
+  const [expanded, setExpanded] = useState(false)
+
+  const controlId = `${uuid}-control`
+
+  return (
+    <div className={cx(styles.root)}>
+      <h3 className={styles.accordionHeader}>
```
Comment (DanielBalsam) [inline src/components/Accordion/Accordion.js:15]: followed the example laid out here w/r/t accessibility: https://www.w3.org/TR/wai-aria-practices-1.1/examples/accordion/accordion.html

File: src/components/Accordion/Accordion.module.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  text-overflow: ellipsis;
+
+  &:focus {
+    outline: 0;
+    box-shadow: 0 0 0 4px white, 0 0 0 6px var(--gray-500);
+  }
+}
+
+.accordionContents {
+  margin: 0px;
```
Comment (DanielBalsam) [inline src/components/Accordion/Accordion.module.scss:47]: right now it doesn't animate. unfortunately would need to use JS to do that. not too hard to do later if we want to

File: src/pages/style-guide.js (hunk 2 above)
Comment (adekunleoduye) [reply src/pages/style-guide.js:86]: I messed up my prettier/vs code setup but this looks good to me.

File: src/helpers/dom.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable no-bitwise */
+export const generateUUID = () => {
+  const crypto = window.crypto || window.msCrypto
```
Comment (adekunleoduye) [inline src/helpers/dom.js:3]: The build is fail due to `"window" is not available during server side rendering`. You will need to add a guard clause for it to work (checkout the [gatsby docs](https://www.gatsbyjs.org/docs/debugging-html-builds/#how-to-check-if-window-is-defined)).

**Comment by hannavas** [comment]
For accordion:

1. Label of accordion (text that says "This is an example accordion") should be in IBM Plex Sans Medium Italic (I spoke to Adekunle about adding that weight to the style guide, let me know if I can explain further)
2. When accordion opens, the plus should change into an x. Let me know if I provide svgs for this, since the + and x are not from the font.
3. Padding around the label (when accordion is closed) should be 24px

For input:
1. Input field should be 51px tall
2. Right and left padding about input label should be 24px
3. Input font should be IBM Plex Mono ("Text" weight) at 16px font size
4. Could we try not having a label? Let me know if there are accessibility concerns or whether we can use aria

**Pull request merged**